docs/site/components/docs/docs.html

```diff
@@ -68,11 +68,11 @@ <h2 ng-if="method.name[0].toUpperCase() === method.name[0]">
         <span class="view-code-link" docs-code-link></span>
       </h3>
       <p ng-if="method.description" ng-bind-html="method.description"></p>
-      <div ng-show="method.name[0].toUpperCase() === method.name[0]"
+      <div ng-show="method.constructor"
```

Comment:
Made these changes so the `module:gcloud` constructor shows the available methods.